interop/Cargo.toml

```diff
@@ -5,7 +5,9 @@ edition = "2021"
 license = "GPL-3.0-only"
 
 [features]
-default = ["proteus"]
+default = []
```

Comment:
having the proteus feature not default is a good thing (for local tests) but you should force it in CI jobs i.e. "cargo run --bin interop --features proteus"

Reply:
I think it's an artifact of running things fast. I'll revert it, it has no reason to be changed in this PR